ra-optimisation/linear-rao/src/main/java/com/farao_community/farao/linear_rao/LinearRao.java

```diff
@@ -56,16 +56,25 @@ public String getVersion() {
                                                        String variantId,
                                                        ComputationManager computationManager,
                                                        RaoParameters parameters) {
+        LinearRaoParameters linearRaoParameters = parameters.getExtensionByName("LinearRaoParameters");
+        if (linearRaoParameters == null) {
+            throw new FaraoException("rao parameters is missing linear rao parameters extension");
+        }
```

Comment:
I would create a configuration quality check, as in the searchTreeRao.

Reply:
ok (only checks if linearRaoParameters exist for now)